docs/usage/configuration-options.md

```diff
@@ -2200,22 +2200,64 @@ e.g.
 
 The `postUpgradeTasks` configuration consists of three fields:
 
-### commands
+### commands (postUpgradeTasks)
 
 A list of commands that are executed after Renovate has updated a dependency but before the commit is made.
 
-You can use variable templating in your commands if [`allowPostUpgradeCommandTemplating`](https://docs.renovatebot.com/self-hosted-configuration/#allowpostupgradecommandtemplating) is enabled.
+You can use variable templating in your commands if [`allowUpgradeCommandTemplating`](https://docs.renovatebot.com/self-hosted-configuration/#allowUpgradeCommandTemplating) is enabled.
 
-### fileFilters
+### fileFilters (postUpgradeTasks)
 
 A list of glob-style matchers that determine which files will be included in the final commit made by Renovate.
 
-### executionMode
+### executionMode (postUpgradeTasks)
 
 Defaults to `update`, but can also be set to `branch`.
 This sets the level the postUpgradeTask runs on, if set to `update` the postUpgradeTask will be executed for every dependency on the branch.
 If set to `branch` the postUpgradeTask is executed for the whole branch.
 
+## preUpgradeTasks
+
+<!-- prettier-ignore -->
+!!! note
+    Pre-upgrade tasks can only be used on self-hosted Renovate instances.
```

Comment:
Once you move the whole section to the self-hosted config option docs, you can drop this admonition. 😉